tests/complete/test_parser.py

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
```

Comment:
Actually, I'm not sure these tests should have moved out from where they were at.
"Complete" means end to end tests; While these are not unit tests per se, they are parser specific.
We already have many tests which aren't standard unit tests yet are treated as such. See execution plugin tests for example - those actually run the workflow engine.
On the other hand, the parser might indeed be used regardless of other components of ARIA by other projects, and so it might be ok to leave this here.
Again, I'll leave it for your consideration.